documentation/docs/11-ssr-and-javascript.md

```diff
@@ -64,13 +62,7 @@ In many cases, you'll only want to prerender specific pages in your app. You'll
 
 The prerenderer will start at the root of your app and generate HTML for any prerenderable pages it finds. Each page is scanned for `<a>` elements that point to other pages that are candidates for prerendering — because of this, you generally don't need to specify which pages should be accessed. If you _do_ need to specify which pages should be accessed by the prerenderer, you can do so with the `pages` option in the [prerender configuration](#configuration-prerender).
 
-#### When not to prerender
-
-The basic rule is this: for a page to be prerenderable, any two users hitting it directly must get the same content from the server.
-
-> In other words, any app that involves user sessions or authentication is _not_ a candidate for `adapter-static`, even if individual pages within an app _are_ suitable for prerendering.
-
-Note that you can still prerender pages that load data based on the page's parameters, like our `src/routes/blog/[slug].svelte` example from earlier. The prerenderer will intercept requests made inside `load`, so the data served from `src/routes/blog/[slug].json.js` will also be captured.
+> Not all pages are suitable for prerendering. Any content that is prerendered will be seen by all users and will not be personalized. For a more in-depth discussion, see [the prerendering section of the introduction](#introduction-prerendering).
```

Comment:
Still sounds limiting. Consider adding:

Any content that is prerendered will be seen by all users and will not be personalized , any personalization will need to performed in the browser.

Reply:
I'll drop the "will not be personalized" instead to keep this from getting too long and repeating the intro content